templates/web/fixmystreet.com/around/postcode_form.html

```diff
@@ -0,0 +1,34 @@
+<div id="front-main">
+    <div id="front-main-container">
+        [% UNLESS possible_location_matches %]
+            [% INCLUDE 'around/intro.html' %]
+        [% END %]
+
+        [%
+            question = c.cobrand.enter_postcode_text || loc('Enter a nearby street name and area');
+        %]
+
+        <form action="[% c.uri_for('/around') %]" method="get" name="postcodeForm" class="js-geolocate postcode-form-test">
+            <label for="pc">[% question %]:</label>
+            <input type="text" name="pc" value="[% pc | html %]" id="pc" size="10" maxlength="200" placeholder="[% tprintf(loc('e.g. ‘%s’ or ‘%s’'), c.cobrand.example_places) %]" class="postcode-form-test__postcode">
+            <a href="[% c.uri_for('/around') %]" id="geolocate_link" class="btn btn--geolocate">[% loc('or locate me automatically') %]</a>
+
+            [%# Form will be submitted via this button when user presses Enter key %]
+            <button type="submit" class="visuallyhidden" aria-hidden="true" tabindex="-1">Go</button>
```

Comment:
Why `button` and not `input`?

Reply:
No reason! Just because I created this element by copying one of the `button`s from a few lines further below – and _they_ were `<button>` elements rather than `<input>`s because then I could have their form value (eg: `browse`) be different from their visual text content (eg: "Show problems").

Can you think of a reason we’d prefer inputs, over buttons, in one or both of these places?

Reply:
Nah, I'm just very old and remember browsers that support input and not button, but they'll all be useless if anyone's still using them.

Reply:
Mmmhmmm. For the last few months I’ve been trying to use `<button>`s wherever I can in forms. Mainly because, unlike `<input>`s, they can have `:before` and `:after` pseudo-children, which is handy for adding icons to buttons without messing around with background positioning and i18n.

And the nice thing about `<button>`s is that they don’t _have_ to live in forms. So they’re a nice, more natively accessible drop-in replacement for the `<a href="#">` that we’d typically attach JavaScript listeners to (although not very often in FMS!).

So then you’ve basically got one element you use _anywhere_ you want some discrete, button-like clickable element. 🎉